packages/components/radio/src/radio.types.ts

```diff
@@ -0,0 +1,26 @@
+export interface RadioProps {
```

Comment:
RadioProps need to extend HTMLInputElement, so things only related to the HTML element itself might be passed here as well